global-chart/templates/hooks.yaml

```
@@ -0,0 +1,114 @@
+    "helm.sh/hook-delete-policy": {{ $command.deletePolicy | default "before-hook-creation" | quote }}
+spec:
+  template:
+    metadata:
+      name: {{ template "global-chart.hookfullname" (merge (dict "hookname" $hook "jobname" $name) $dot) }}
+      labels:
+        {{- include "global-chart.labels" $dot | nindent 8 }}
+    spec:
+      {{- with $dot.Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- range . }}
+          {{- if kindIs "string" . }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 10

(indentation)